src/components/Account/AccountPopup.tsx

```diff
@@ -107,10 +107,11 @@ const SubscribedAccountPopup = withSubscription({
   checkForUpdate: (oldProps, newProps) => { return oldProps.accountAddress !== newProps.accountAddress || oldProps.dao.address !== newProps.dao.address; },
 
   createObservable: (props: IProps) => {
+    const subscribe = props.historyView !== true;
```

Comment:
@jellegerbrandy  To me, this syntax is ugly and hard to interpret.  `!props.historyView` is far more common syntax and natural to interpret.